core/src/test/java/com/sap/ai/sdk/core/client/DeploymentUnitTest.java

```diff
@@ -195,4 +208,61 @@ void testDeleteAiDeployment() {
     // cloudSdkCustomFields
```

Comment:
There are leftover `assertThat(response).isNotNull();` in existing tests

Reply:
There is different opinions on whether to keep the assertion for null check. I though I would keep it for now. But, I ll make them all consist.

Reply:
My opinion is: It can NEVER be `null` since it's a unit test. I added those originally to remove IDE warnings since the endpoints were `@Nullable` before.

Reply:
I didn't know about this discussion when I suggested to keep nonnull-check.

I'm aware nonnull is enforced anyway when otherwise a nullpointerexception would be thrown. However I like the nonnull check, as I think they protect us from future changes in our productive code or in our test resources. Not a strong opinion though. I'm fine with either choice.

Reply:
notnull is now enforced in every test.